index.html

```diff
@@ -278,6 +301,7 @@
             },
 
             _onSearch(e) {
+                //
```

Comment:
Add a debounce function here. This is important if the person array contains many thousands of objects instead of a few hundred. Front-end filtering can slow down significantly since the onSearch function executes onChange.